### PR TITLE
Update version with semver

### PR DESCRIPTION
**Pull request by udda1996**
## Purpose
Update version with SemVer and remove short version

### CODE REVIEW

**Review by keizer619 — DISMISSED** [review]

File: gradle.properties [hunk 1]
```diff
@@ -2,10 +2,9 @@ org.gradle.caching=true
 org.gradle.parallel=true
 org.gradle.jvmargs='-Dfile.encoding=UTF-8'
 org.gradle.workers.max=3
-version=2.0.0-beta.7-SNAPSHOT
+version=-2201.0.0-SNAPSHOT
 group=org.ballerinalang
 bootstrappedOn=1.1.0-alpha
-shortVersion=slbeta7
 versionDisplayText=Beta 7
```
Comment (pubudu91) [inline gradle.properties:8]: What about this one?

File: gradle.properties [hunk 2]
```diff
@@ -2,11 +2,9 @@ org.gradle.caching=true
 org.gradle.parallel=true
 org.gradle.jvmargs='-Dfile.encoding=UTF-8'
 org.gradle.workers.max=3
-version=2.0.0-beta.7-SNAPSHOT
+version=-2201.0.0-SNAPSHOT
```
Comment (azinneera) [inline gradle.properties:5]: Why do we have a hyphen (`-`) at the beginning of the version?

**Comment by azinneera** [comment]
The failing test ` io.ballerina.projects.FileSystemRepositoryValidationTests.testGetPackageVersions` can be fixed only after the version update. Please disable the test and go ahead. Will send the fix and enable the test afterwards.

**Comment by udda1996** [comment]
> The failing test ` io.ballerina.projects.FileSystemRepositoryValidationTests.testGetPackageVersions` can be fixed only after the version update. Please disable the test and go ahead. Will send the fix and enable the test afterwards.

ACK. Will disable the test

**Comment by udda1996** [comment]
Screenshot 2021-12-17 at 12 04 45
This is the `bal -v ` output with this fix.

**Comment by udda1996** [comment]
This is how the "bal -v" output looks with this fix.
Screenshot 2021-12-17 at 12 16 11

**Review by keizer619 — APPROVED** [review]

**Pull request merged**